service/src/main/java/uk/gov/hmcts/reform/fpl/utils/NotifyAttachedDocumentLinkHelper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONObject;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.Optional;
+
+import static org.apache.commons.lang3.ArrayUtils.isNotEmpty;
+import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
+import static uk.gov.service.notify.NotificationClient.prepareUpload;
+
+@Slf4j
+public class NotifyAttachedDocumentLinkHelper {
```

Comment:
if we're adding new helper can we also make use of it in other places where we attach a document? OrderIssuedEmailContentProvider + CaseManagementOrderEmailContentProvider can use this helper